src/commands/assets/registerAsset.ts

```diff
@@ -131,16 +125,21 @@ export const registerAsset = async (
 
   logger.info(chalk.dim('\nCreating Asset ...'))
 
+  const assetAttributes = AssetAttributes.getInstance({
+    metadata: ddoMetadata,
+    price: assetPrice,
+    providers: [config.nvm.neverminedNodeAddress!]
+  })
+
+  let publishMetadata = PublishMetadata.OnlyMetadataAPI
+
+  if (argv.publishMetadata && argv.publishMetadata.toLowerCase() === 'ipfs')
```

Comment:
```suggestion
  if (argv.publishMetadata?.toLowerCase() === 'ipfs')
```
it can be reduced to this